charts/opentelemetry-operator/examples/default/rendered/admission-webhooks/operator-webhook-with-cert-manager.yaml

```diff
@@ -14,7 +14,7 @@ webhooks:
     clientConfig:
       service:
         name: opentelemetry-operator-webhook-service
-        namespace: default
+        namespace: monitoring
```

Comment:
Why the namespace changed?

Reply:
that was `kubens` playing tricks on me, that should be fixed